test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml

```diff
@@ -227,6 +227,13 @@
  	<output regex="no" type="failure">Test ran to completion</output>
  	<output regex="no" type="success">java.lang.OutOfMemoryError</output>
  </test>
+ <test id="Excessive GC throws OOM on RISC-V">
+ 	<echo value="CMVC 148977:  Excessive GC throws OOM may fail if there is other activity on the machine - at least 5 GCs are required, in the 20 second test" />
+ 	<!-- does the test fail to run (thus implying that we encountered excessive GC)? -->
+ 	<command>$EXE$ $XINT$ $ARGS_FOR_ALL_TESTS$ -Xdump:none -Xms2m -Xmx2m -Xgc:fvtest=forceExcessiveAllocFailureAfter=5 -verbose:gc -Xverbosegclog:foo.log $CP$ com.ibm.tests.garbagecollector.SpinAllocate 20</command>
+ 	<output regex="no" type="failure">Test ran to completion</output>
+ 	<output regex="no" type="success">java.lang.OutOfMemoryError</output>
+ </test>
```

Comment:
I am not sure we need special version for RISC-V but just replace `-Xms8m -Xmx8m` to `-Xms2m -Xmx2m` for general. @amicic Any complains about this?

Reply:
Don't know. Perhaps there is a danger to hit genuine  OOM before excessive OOM with 2M, although I don't have a valid theory that it could  occur on other platforms and not on RISC.

Reply:
ok, thank you. @ChengJin01 Would you check that `-Xms2m -Xmx2m` works with this test on all platforms?

Reply:
Will launch personal builds to verify after updating all settings.

Reply:
ok, thank you. If we can not apply new settings for all platforms so you can limit it to RISC-V only as proposed